packages/build/tests/error/fixtures/plugin_homepage/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "test-plugin",
   "description": "test",
   "version": "1.0.0",
+  "type": "module",
```

Comment:
Many test fixtures' `package.json` now use `"type": "module"`. In most cases, this is not needed since `@netlify/build`'s `package.json` already has it. However, we add it for consistency, and also so that those `package.json` are semantically more valid. For example, if the fixture directory was to be copied to a temporary directory in the future, `"type": "module"` would then be needed.